tests/discover/options.sh

```diff
@@ -8,16 +8,44 @@ rlJournalStart
         rlRun 'set -o pipefail'
     rlPhaseEnd
 
-    rlPhaseStartTest 'fmf help'
-        rlRun 'tmt run discover --help --how fmf | tee output'
-        rlAssertGrep 'Discover available tests from fmf metadata' 'output'
+    # Note tmt has different levels of '--help', and L1 > L2 > L3 > L4
+    #
+    #     ----     ------     ------          ------           ------
+    #     #ID#     #L1        #L2             #L3              #L4
+    #     ----     ------     ------          ------           ------
+    #      1)  tmt --help run --help discover --help --how fmf --help
+    #      2)  tmt        run --help discover --help --how fmf --help
+    #      3)  tmt        run        discover --help --how fmf --help
+    #      4)  tmt        run        discover        --how fmf --help
```

Comment:
Checking the different `--help` levels is not closely related to the `discover` step. What about moving this four-level coverage to `/tests/core/docs`?

https://github.com/teemtee/tmt/blob/c6f68a1ebee3dbb625fef80f13c150c170e36f69/tests/core/docs/test.sh#L22-L25

Reply:
@psss, the related tests are updated according to your suggestion, please help to take a look again, thanks!